docs/CARS.md

```diff
@@ -6,216 +6,216 @@ A supported vehicle is one that just works when you install a comma three. All s
 
 # 208 Supported Cars
 
-|Make|Model|Supported Package|ACC|No ACC accel below|No ALC below|Steering Torque|Resume from stop|Harness|
```

Comment:
are you just running ./selfdrive/cars/docs.py? make sure your IDE doesn't touch this file